### ISSUE TITLE
Multilabel support

### ISSUE THREAD

**Issue by mpangrazzi** [huggingface/setfit#65]
Hi! 

do you think it could be possible to handle multilabel classification as well (and so handling an array of labels for each single example)? Like e.g. wrapping the LogisticRegression using an [OneVsRestClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.multiclass.OneVsRestClassifier.html)?

Thanks!

PS: this is a very nice work! 😄

**Comment by lewtun** [comment]
Hey @mpangrazzi, yes SetFit works very well in multi-label scenarios and @fgbelidji has implemented a script here for that: https://github.com/huggingface/setfit/blob/b5da11157b85ef68f01b87dc1058fa36f36ba291/scripts/setfit/run_fewshot_multilabel.py#L104

What's currently missing is a nice integration with the `SetFitTrainer`, so a PR for that would be very welcome (if I don't get to it first :))

**Comment by sanderland** [comment]
Also very interested in trying this on multi-label cases. 
One question from reading the code: am I correct in understanding `sentence_pairs_generation_multilabel` gives a label "1" if any label overlaps and "0" on completely disjoint examples? If so, doesn't this give problems when e.g. one label is a strict subset of another, or one label is rather unbalanced.

**Comment by mpangrazzi** [comment]
Added in #69
